src/bin/kafka_consumer.rs

```diff
@@ -47,6 +47,10 @@ struct Cli {
     #[arg(long, value_name = "MAX", default_value_t = 15000)]
     max_in_queue: usize,
 
+    /// Simulated mode (for testing purposes, LSST only)
+    #[arg(long, default_value_t = false)]
+    simulated: bool,
+
```

Comment:
Nit: Consider `BOOM_SIMULATE_LSST` env var or something for config in lieu of a command line arg.

Reply:
I agree. It's a minor thing--the logic being that an uncommon "dev" option is better slightly hidden, but it's only ever going to be run by devs or an automated process.